core/tests/test_views.py

```diff
@@ -154,27 +155,26 @@ def test_terms_conditions_cms_disabled(
     'view,expected_template',
     (
         (
-            'terms-and-conditions',
-            'core/terms_conditions-domestic-cms.html'
+            'privacy-and-cookies',
+            'core/privacy_cookies-domestic-cms.html'
         ),
         (
-            'terms-and-conditions-international',
-            'core/terms_conditions-international-cms.html'
+            'privacy-and-cookies-international',
+            'core/privacy_cookies-international-cms.html'
         ),
     )
 )
-@patch('core.views.cms_client.export_readiness.get_terms_and_conditions_page')
-def test_terms_conditions_cms_enabled(
-    mock_get_t_and_c_page, view, expected_template, client, settings
+@patch('core.views.cms_client.export_readiness.get_privacy_and_cookies_page')
+def test_privacy_cookies_cms(
```

Comment:
it looks like this has replaced the terms and conditions test?

Reply:
t&c is still up `test_terms_conditions_cms`